<ui version="4.0" >
 <class>qmvdbconnectClass</class>
 <widget class="QDialog" name="qmvdbconnectClass" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>320</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>288</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>288</width>
    <height>320</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="3" >
    <widget class="QLabel" name="panelLabel" >
     <property name="font" >
      <font>
       <family>Sans Serif</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text" >
      <string>Configure Database Connections</string>
     </property>
     <property name="textFormat" >
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="labelLabel" >
     <property name="text" >
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QLineEdit" name="labelEdit" />
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="nameLabel" >
     <property name="toolTip" >
      <string>Database Name</string>
     </property>
     <property name="statusTip" >
      <string>Database Name</string>
     </property>
     <property name="whatsThis" >
      <string>The name of the database</string>
     </property>
     <property name="text" >
      <string>Database</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QLineEdit" name="nameEdit" />
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="hostLabel" >
     <property name="toolTip" >
      <string>Database Hostname</string>
     </property>
     <property name="statusTip" >
      <string>Database Hostname</string>
     </property>
     <property name="whatsThis" >
      <string>The Hostname or IP address of the database host</string>
     </property>
     <property name="text" >
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" >
    <widget class="QLineEdit" name="hostEdit" />
   </item>
   <item row="4" column="0" >
    <widget class="QLabel" name="portLabel" >
     <property name="text" >
      <string>Port</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" >
    <widget class="QLineEdit" name="portEdit" />
   </item>
   <item row="5" column="0" >
    <widget class="QLabel" name="userLabel" >
     <property name="toolTip" >
      <string>User Name</string>
     </property>
     <property name="statusTip" >
      <string>User Name</string>
     </property>
     <property name="whatsThis" >
      <string>The Hostname or IP address of the database host</string>
     </property>
     <property name="text" >
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QLineEdit" name="userEdit" />
   </item>
   <item row="6" column="0" >
    <widget class="QLabel" name="passwordLabel" >
     <property name="toolTip" >
      <string>User Password</string>
     </property>
     <property name="statusTip" >
      <string>User Password</string>
     </property>
     <property name="whatsThis" >
      <string>The Hostname or IP address of the database host</string>
     </property>
     <property name="text" >
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QLineEdit" name="passwordEdit" >
     <property name="echoMode" >
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0" >
    <widget class="QLabel" name="optionsLabel" >
     <property name="toolTip" >
      <string>Connection Options</string>
     </property>
     <property name="statusTip" >
      <string>Connection Options</string>
     </property>
     <property name="whatsThis" >
      <string>Additional connection options</string>
     </property>
     <property name="text" >
      <string>Options</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" >
    <widget class="QLineEdit" name="optionsEdit" />
   </item>
   <item row="8" column="0" >
    <widget class="QPushButton" name="testButton" >
     <property name="whatsThis" >
      <string>Test the connection to the database</string>
     </property>
     <property name="text" >
      <string>&amp;Test</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2" >
    <widget class="QLabel" name="connectLabel" >
     <property name="toolTip" >
      <string>Connection Status</string>
     </property>
     <property name="statusTip" >
      <string>Connection Status</string>
     </property>
     <property name="whatsThis" >
      <string>Status of database connection</string>
     </property>
     <property name="text" >
      <string>Not Connected</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2" >
    <widget class="QDialogButtonBox" name="exitControls" >
     <property name="toolTip" >
      <string/>
     </property>
     <property name="statusTip" >
      <string/>
     </property>
     <property name="whatsThis" >
      <string/>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Discard|QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qdocman.qrc" />
 </resources>
 <connections/>
</ui>
